addon/serializers/rest-serializer.js

```diff
@@ -29,6 +32,64 @@ export default ActiveModelSerializer.extend({
 
   getCoalescedIds(request) {
     return request.queryParams && request.queryParams.ids;
-  }
+  },
+
+  /**
+   * @method _maybeAddAssociationIds
+   * @param model
+   * @param attrs
+   * @private
+   */
+  _maybeAddAssociationIds(model, attrs) {
+    let newHash = _assign({}, attrs);
+
+    if (this.serializeIds === 'always') {
+      model.associationKeys.forEach((key) => {
+        let association = model[key];
+        if (this.isCollection(association)) {
+          let formattedKey = this.keyForRelationshipIds(key);
+          newHash[formattedKey] = model[key].models.map((m) => {
+            return { id: m.id, type: m.modelName };
```

Comment:
This is copy and paste of the function here: https://github.com/samselikoff/ember-cli-mirage/blob/master/addon/serializer.js#L279 with only this line changed. I didn't really know the best way to just change this line.